From 2328d5295d35a373232210cf8477d92856423808 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Wed, 6 Nov 2019 13:58:02 +0100
Subject: [PATCH 03/61] build: use python3 to run landmines

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index fc2d94d4ce37b..18896d7e57949 100644
--- a/DEPS
+++ b/DEPS
@@ -3421,7 +3421,7 @@ hooks = [
     'name': 'landmines',
     'pattern': '.',
     'action': [
-        'python',
+        'python3',
         'src/build/landmines.py',
     ],
   },
-- 
2.26.0

