From c39e9fe4a81e16f1213fbbf56b8c0f68cd95d050 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 25 Sep 2017 20:52:08 +0200
Subject: [PATCH 07/66] build: fix more ucontext stupidity

---
 base/logging_unittest.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/logging_unittest.cc b/base/logging_unittest.cc
index 0a1fed40830a..d8ad4005836c 100644
--- a/base/logging_unittest.cc
+++ b/base/logging_unittest.cc
@@ -288,7 +288,7 @@ void CheckCrashTestSighandler(int, siginfo_t* info, void* context_ptr) {
 #if defined(OS_MACOSX)
   crash_addr = reinterpret_cast<uintptr_t>(info->si_addr);
 #else  // OS_POSIX && !OS_MACOSX
-  struct ucontext* context = reinterpret_cast<struct ucontext*>(context_ptr);
+  ucontext_t* context = reinterpret_cast<ucontext_t*>(context_ptr);
 #if defined(ARCH_CPU_X86)
   crash_addr = static_cast<uintptr_t>(context->uc_mcontext.gregs[REG_EIP]);
 #elif defined(ARCH_CPU_X86_64)
-- 
2.14.2

