From 152369492043932fa58f2d4c065447a1e9aa2fe0 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 7 Nov 2019 08:03:36 +0100
Subject: [PATCH 08/60] [openSUSE chromium-77-clang.patch]

---
 base/location.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/location.h b/base/location.h
index c07e747fe8f75..924db1cb818fe 100644
--- a/base/location.h
+++ b/base/location.h
@@ -18,7 +18,7 @@
 
 namespace base {
 
-#if defined(__has_builtin)
+#if defined(__clang__)
 // Clang allows detection of these builtins.
 #define SUPPORTS_LOCATION_BUILTINS                                       \
   (__has_builtin(__builtin_FUNCTION) && __has_builtin(__builtin_FILE) && \
-- 
2.23.0

