From 1e6f97a7d8419446cae0012e73689839eca26665 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Wed, 8 Apr 2020 21:44:57 +0200
Subject: [PATCH 09/61] openSUSE chromium-81-gcc-constexpr.patch

---
 ui/views/layout/layout_types.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/views/layout/layout_types.h b/ui/views/layout/layout_types.h
index 8f9dbf4faad62..ba118b92e88ad 100644
--- a/ui/views/layout/layout_types.h
+++ b/ui/views/layout/layout_types.h
@@ -45,12 +45,12 @@ class VIEWS_EXPORT SizeBounds {
   ~SizeBounds() = default;
 
   constexpr const base::Optional<int>& width() const { return width_; }
-  constexpr void set_width(base::Optional<int> width) {
+  inline void set_width(base::Optional<int> width) {
     width_ = std::move(width);
   }
 
   constexpr const base::Optional<int>& height() const { return height_; }
-  constexpr void set_height(base::Optional<int> height) {
+  inline void set_height(base::Optional<int> height) {
     height_ = std::move(height);
   }
 
-- 
2.26.0

